backend/Functions/Edna.AssignmentLearnContent/Edna.AssignmentLearnContent/AssignmentLearnContentApi.cs

```diff
@@ -114,9 +173,12 @@ public AssignmentLearnContentApi(ILogger<AssignmentLearnContentApi> logger, IMap
             IList<TableResult> executeBatchResults = await assignmentLearnContentTable.ExecuteBatchAsync(deleteBatchOperations);
             bool errorExists = executeBatchResults.Any(result => result.HttpStatusCode < 200 || result.HttpStatusCode >= 300);
 
-            return errorExists 
-                ? (IActionResult) new InternalServerErrorResult() 
-                : new OkResult();
+            if (errorExists)

```

Comment:
The existing return seems to capture the intent succinctly. Request to please provide your POV for changing it.

Reply:
changed it to add the extra logging, reverting my changes as per your view that we don't require additional logging